samples/code-server/README.md

```diff
@@ -0,0 +1,48 @@
+# [VSCODE web based](https://github.com/coder/code-server) with jupyter
+
+## Install vscode
+
+> Run in jupyter terminal
+
+```bash
+curl -fsSL https://code-server.dev/install.sh | sh
+```
+
+## Setting up server parameters
+
+> In the server config, you need to add the following lines
+
+**jupyter_lab_config.py**
+
+```py
+from tornado import websocket
+websocket_max_message_size = 1048 * 1024 * 1024
+# fix module
+setattr(websocket, "_default_max_message_size", websocket_max_message_size)
```

Comment:
Is WebSocket max size in the tornado setting not enough?

Reply:
You see, there is 10 MB by default.
Which for example for plotly is extremely small. "doesn't fit".
This caused quite a lot of bugs in my work and server crashes. Both jupyter and vscode. Therefore, expanding this parameter is important.

Reply:
I understand, but setting it with `c.LabApp.tornado_settings = {"websocket_max_message_size": websocket_max_message_size}` should be enough, I think you don't need to patch the `tornado.websocket` module.

Reply:
before you use `c.NotebookApp`, which won't work for JupyterLab

Reply:
ah I see, could you add this link to your comment? It could help other users.